src/cmds/qselect.c

```diff
@@ -72,55 +82,53 @@ set_attrop(struct attropl **list, char *a_name, char *r_name, char *v_name, enum
 {
 	struct attropl *attr;
 
-	attr = (struct attropl *) malloc(sizeof(struct attropl));
+	attr = (struct attropl *)malloc(sizeof(struct attropl));
 	if (attr == NULL) {
 		fprintf(stderr, "qselect: out of memory\n");
 		exit(2);
 	}
-	if (a_name == NULL)
+
+	if (a_name == NULL) {
 		attr->name = NULL;
-	else {
-		attr->name = (char *) malloc(strlen(a_name)+1);
+	} else {
+		attr->name = (char *)malloc(strlen(a_name)+1);
```

Comment:
I've been seeing several places where we do malloc/strncpy rather than strdup. I suggest you change these to strdup.